.github/workflows/publish.yaml

```
@@ -32,9 +32,10 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Upload component to the component registry
-      uses: espressif/upload-components-ci-action@v1
+      uses: espressif/upload-components-ci-action@v2
       with:
-        name: "astarte-device-sdk-esp32"
+        components: "astarte-device-sdk-esp32: . "
```

Is it right ": ." at the end?
```suggestion
        components: "astarte-device-sdk-esp32"
```

---

Yes, it's the new syntax of the action. It's `name-of-the-component`:`path to the component`
See [here](https://github.com/espressif/upload-components-ci-action)